packages/app/src/device-log/controllers/device-log.controller.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { Controller, IpcEvent } from "App/core/decorators"
+import { ResultObject } from "App/core/builder"
+import { DeviceLogService } from "App/device-log/services"
+import { ControllerPrefix, IpcDeviceLogEvent } from "App/device-log/constants"
+import { DeviceFile } from "App/device-file-system/dto"
+import { DeviceFilesOption } from "App/device-file-system/types"
+
+@Controller(ControllerPrefix)
+export class DeviceLogController {
+  constructor(private deviceInfoService: DeviceLogService) {}
+
+  @IpcEvent(IpcDeviceLogEvent.GetLog)
+  public async getDeviceInfo(
```

Comment:
Name seems to be invalid

Reply:
https://github.com/mudita/mudita-center/pull/1042/commits/fadc6b05c98ff4e9f1fcc35d49fb082cb9643cab